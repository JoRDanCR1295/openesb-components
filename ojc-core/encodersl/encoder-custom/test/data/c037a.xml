<?xml version="1.0" encoding="UTF-8"?>
<root xmlns="http://xml.netbeans.org/schema/test">
    <groupNode1>
        <element1>
            <delimField1>a</delimField1>
        </element1>
        <element2>
            <fixedField1>0123456789</fixedField1>
        </element2>
    </groupNode1>
</root>
