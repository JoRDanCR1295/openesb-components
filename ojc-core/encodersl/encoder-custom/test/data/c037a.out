a,the_default_value|0123456789defval         